test/integration/pkg/generated/openapi_generated.go

```diff
@@ -1,3 +1,5 @@
+// +build !ignore_autogenerated
```

Comment:
This file used to be kept mostly empty. I honestly believe this is a bug, it shouldn't have to be committed at all.

Reply:
I have similar feeling. I don't know the reason for committing this file at all.

I first ran the commands in README.md and got the file updated like in this PR.

When I then ran `go test -v .` to test locally, the file was updated again: 
```
-// Code generated by openapi-gen. DO NOT EDIT.
+// Code generated by openapi-gen.go. DO NOT EDIT.
```

I didn't look deep into it because it doesn't fail the test. Maybe we should just have the integration test remove the file as cleanup, and also add a command in README.md to remove the file, so that it won't be committed. WDYT?